index.html

```diff
@@ -49,10 +49,193 @@ <h2>LET'S CONNECT</h2>
               <a href="https://github.com/isaac-ej"><img src="img/Vector (1).svg" alt=""></a>
             </li>
           </ul>
-        
         </section>
-   </header>
+     </header>
+       <main class="card-container">
+         <!-- Work section -->
+         <section class="container">
+          <div class="bg">
+            <div class="snap">
+              <img src="/img/Snapshoot Portfolio.png">
+            </div>
+            <div class="left-block">
+              <div class="primary-text">
+                <h5 class="tonic">TONIC </h5>
+                <div class="frame">
+                  <p>Canopy</p>
+                  <ul>
+                    <li><div class="spans"></div></li>
+                    <li>Backend Dev</li>
+                    <li><p class="spans"></p></li>
+                    <li>2015</li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+            <div class="techs">
+              <ul>
+                <li>HTML</li>
+                <li> CSS</li>
+                <li>Javascript</li>
+              </ul>
+            </div>
+            <button>See Projects</button>
+          </div>
+         </section>
+
+        <section class="container">
+          <div class="bg">
+            <div class="snap">
+              <img src="/img/Snapshoot Portfolio(2).png">
+            </div>
+            <div class="left-block">
+              <div class="primary-text">
+                <h5 class="tonic">Uber Navigation</h5>
+                <div class="frame">
+                  <p>Canopy</p>
+                  <ul>
+                    <li>
+                      <div class="spans"></div>
+                    </li>
+                    <li>Backend Dev</li>
+                    <li>
+                      <p class="spans"></p>
+                    </li>
+                    <li>2015</li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+            <div class="techs">
+              <ul>
+                <li>HTML</li>
+                <li> CSS</li>
+                <li>Javascript</li>
+              </ul>
+            </div>
+            <button>See Projects</button>
+          </div>
+        </section>
+
+        <section class="container">
```

Comment:
- [Optional] This is what your work card looks like on the 375px screensize. Kindly fix the highlighted section to match the figma template.
### Your design
![image](https://github.com/Isaac-ej/Portfolio_Website/assets/94551484/054c643b-b0ef-4049-bf94-74a954ee7b32)
### Figma
![image](https://github.com/Isaac-ej/Portfolio_Website/assets/94551484/7a1e957b-a2fc-4bf3-970f-1bb08fc5c9f3)

Reply:
- [ ] Kindly implement the required changes requested by the previous reviewer 🙏🏽 
Following the design from the Figma template as much as possible would demonstrate your attention to details skill, which is a **highly** sought-after skill by recruiters 👁️ 👁️